orchestration/dagster_orchestration/hca_manage/validation.py

```diff
@@ -0,0 +1,65 @@
+from google.cloud import storage
+import logging
+from jsonschema import validate
+import json
+import requests
+from typing import Optional
+from hca_manage.common import DefaultHelpParser
+
+
+def validate_json(blob: storage.Blob) -> Optional[Exception]:
+    """
+    Validate that the JSON file blob follows the schema in the describedBy
+    :param blob: JSON file blob to check
+    :return: An exception if JSON is invalid, return None if valid
+    """
+    try:
+        data = json.loads(blob.download_as_string(client=None))
```

Comment:
⛏️ Seems like `client=None` is redundant, can you just call it without the optional kwarg? ie.. 

```suggestion
        data = json.loads(blob.download_as_string())
```